tests/std/tests/VSO_0000000_allocator_propagation/test.cpp

```diff
@@ -946,6 +946,20 @@ _CONSTEXPR20 void test_string_swap(const size_t id1, const size_t id2) {
     assert(dst.get_allocator().id() == id1);
 }
 
+#if _HAS_CXX20

```

Comment:
This don't have to be guarded by `_HAS_CXX20`, as `strbuf(move(str))` will still call copy constructor in pre-cpp20 mode. However, I think adding `_HAS_CXX20` will make it more obvious that the leak is relevant to `basic_stringbuf(basic_string&&,...)` introduced in C++20.